app/common/gristUrls.ts

```diff
@@ -995,76 +1005,48 @@ export function isClient() {
   return (typeof window !== 'undefined') && window && window.location && window.location.hostname;
 }
 
+function getCustomizableValue(
+  clientSideConfigKey: keyof GristLoadConfig,
+  serverSideEnvVar: keyof NodeJS.ProcessEnv
+) {
+  return isClient() ? (window as any).gristConfig?.[clientSideConfigKey] : process.env[serverSideEnvVar];
```

Comment:
Am I understanding this correctly that `getHelpCenterUrl()` could return a different value on the server compared to the client, since on the server it won't respect something set in `GRIST_CUSTOM_COMMON_URLS`?